MyOwnORM/MyOwnORM/Services/CustomDbSetService.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using MyOwnORM.Reflection;
+
+namespace MyOwnORM
+{
+    public class CustomDbSetService<T> where T : class
+    {
+        private readonly string _connectionString;
+        private CustomDbSetReflection<T> dbSetReflection;
+        public CustomDbSetService(string connectionString)
+        {
+            _connectionString = connectionString;
+        }
+        public async Task<List<string>> FindTablesNameWithForeignKeysForCascadeDelete(string tableName)
```

Comment:
Since you have deleted CascadeDeleteAttribute, if I understood correctly, each entity's property will be deleting cascadely?

Reply:
Yes